src/main/java/use_case/change_password/ChangePasswordInteractor.java

```diff
@@ -22,7 +22,8 @@ public ChangePasswordInteractor(ChangePasswordUserDataAccessInterface changePass
     @Override
     public void execute(ChangePasswordInputData changePasswordInputData) {
         final User user = userFactory.create(changePasswordInputData.getUsername(),
-                                             changePasswordInputData.getPassword());
+                                             changePasswordInputData.getPassword(),
+                                             changePasswordInputData.getEmail());
```

Comment:
Since email and username would never change, make sure that there is no way it is reset as it would affect all posts and profile retrieval.